bundles/framework/publisher2/resources/scss/style.scss

```diff
@@ -23,6 +23,10 @@
         }
     }
 }
+      
+#publisher-announcements {
```

Comment:
Don't use ids when you don't need to. And shouldn't this be inside `div.basic_publisher`?